README.md

```diff
@@ -88,5 +88,5 @@ npm run test
 
 ### Requirements
 
-- [Node 20 or higher](https://nodejs.org/en/download/)
-- [NPM 10 or higher](https://www.npmjs.com/package/npm)
+-   [Node 20 or higher](https://nodejs.org/en/download/)
```

Comment:
Seems strange but enforced by prettier I assume?

Reply:
Yes to allow aligning with tab width when having multi-line list points:
without:

```md
- foo
    bar
```

with:

```md
-   foo
    bar
```